generator.yaml

```diff
@@ -621,6 +621,56 @@ resources:
         from:
           operation: DescribeFeatureGroup
           path: FeatureGroupStatus
+  NotebookInstance:
+    exceptions:
+      errors:
+          404:
+            code: ValidationException
+            message_prefix: RecordNotFound
+      terminal_codes:
+        - ResourceLimitExceeded
```

Comment:
Have you reviewed this with Kartik or Suraj offline ? there is a comment about it in the document, can you make sure to update the doc accordingly and resolve the comment.